### PR TITLE
#94 Headers must be strings

### PR DESCRIPTION
**Pull request by rudolfbyker**

### CODE REVIEW

File: MiscVBAFunctions/Modules/MiscArray.bas [hunk 1]
```diff
@@ -36,6 +40,13 @@ Public Function ArrayToRange( _
     EndRow = StartRow + UBound(Data) - LBound(Data)
     EndColumn = StartColumn + UBound(Data, 2) - LBound(Data, 2)
     
+    If IncludesHeader Then
```
Comment (malankriel) [inline MiscVBAFunctions/Modules/MiscArray.bas:43]: Why not do this in `ArrayToNewTable` and remove this additional argument from `ArrayToRange`?

File: MiscVBAFunctions/Modules/Test__MiscArray.bas [hunk 2]
```diff
@@ -244,6 +244,46 @@ TestFail:
...
+    Dim Arr(1, 3) As Variant
+    Dim RangeObj As Range
+    Dim RangeOutput As Range
+    
+    'Act:
+    Set WB = ExcelBook("", False, False)
+    Arr(0, 0) = "asdf"
+    Arr(0, 1) = 1234
+    Arr(0, 2) = "2022/11/02"
+    Arr(0, 3) = False
```
Comment (malankriel) [inline MiscVBAFunctions/Modules/Test__MiscArray.bas:262]: What would happen with "False"? Stay `False` in the header? :)

**Comment by malankriel** [comment]
Looks good!

File: MiscVBAFunctions/Modules/Test__MiscArray.bas (hunk 2 above)
Comment (rudolfbyker) [reply MiscVBAFunctions/Modules/Test__MiscArray.bas:262]: It becomes the string "FALSE"

File: MiscVBAFunctions/Modules/MiscArray.bas (hunk 1 above)
Comment (rudolfbyker) [reply MiscVBAFunctions/Modules/MiscArray.bas:43]: Two reasons:

- Less code duplication if we use ArrayToRange for something else where the first row is also considered to be a header, e.g. for creating a Named Range table.
- The ranges are already calculated in ArrayToRange, so this keeps similar logic together, which is easier to maintain.

**Pull request merged**